pkg/lclient/storage.go

```diff
@@ -0,0 +1,41 @@
+package lclient
+
+import (
+	"reflect"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	volumeTypes "github.com/docker/docker/api/types/volume"
+	"github.com/pkg/errors"
+)
+
+// CreateVolume creates a Docker volume with the given labels and the default Docker storage driver
+func (dc *Client) CreateVolume(labels map[string]string) (types.Volume, error) {
+	volume, err := dc.Client.VolumeCreate(dc.Context, volumeTypes.VolumeCreateBody{
+		Driver: DockerStorageDriver,
+		Labels: labels,
+	})
```

Comment:
Do we also want to keep the volume name unique by not passing the name and only relying on labels?
```
Maysuns-MacBook-Pro:odo maysun$ d volume ls
DRIVER              VOLUME NAME
local               b8e4b86d235b571354b3e2080b1eb07f05cc5234746cc1f8ca0b182fd228d9f5
local               codewind_cw-workspace
Maysuns-MacBook-Pro:odo maysun$ d volume inspect b8e4b86d235b571354b3e2080b1eb07f05cc5234746cc1f8ca0b182fd228d9f5
[
    {
        "CreatedAt": "2020-04-10T17:02:20Z",
        "Driver": "local",
        "Labels": {
            "component": "myspring",
            "type": "projects"
        },
        "Mountpoint": "/var/lib/docker/volumes/b8e4b86d235b571354b3e2080b1eb07f05cc5234746cc1f8ca0b182fd228d9f5/_data",
        "Name": "b8e4b86d235b571354b3e2080b1eb07f05cc5234746cc1f8ca0b182fd228d9f5",
        "Options": null,
        "Scope": "local"
    }
]
```

Reply:
We should still rely on labels to be safe, as we can let Docker deal with any potential naming conflicts.